src/main/java/org/icgc/argo/program_service/model/entity/Cancer.java

```diff
@@ -0,0 +1,45 @@
+package org.icgc.argo.program_service.model.entity;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.icgc.argo.program_service.model.enums.JavaFields;
+import org.icgc.argo.program_service.model.enums.SqlFields;
+import org.icgc.argo.program_service.model.enums.Tables;
+import org.icgc.argo.program_service.model.join.ProgramCancer;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.google.common.collect.Sets.newHashSet;
+
+
+@Entity
+@Table(name = Tables.CANCER)
+@Builder
+@Data
+@EqualsAndHashCode(of = "name")
```

Comment:
Please replace this with `@EqualsAndHashCode.Exclude` as shown in the [EqualsAndHashCode lombok documentation](https://projectlombok.org/features/EqualsAndHashCode)
Likewise with `@ToString` annotation.